### PR TITLE
Replace webpack with vite

### PR DESCRIPTION
**Pull request by kwingram25**
- Use vite instead of webpack
- Improved HMR and reloading
- Remove textile hub - unused library

closes https://github.com/paritytech/contracts-ui/issues/267

### CODE REVIEW

**Comment by kwingram25** [comment]
Unclear why the deploy preview is broken, may need to adjust netlify settings?

**Comment by statictype** [comment]
please also change the commands in the `build` and `deploy` github jobs.

File: src/db/util/init.ts [hunk 1]
```diff
@@ -89,27 +82,3 @@ export async function initIdentity(db: DB): Promise<[UserDocument | null, Privat
...
-
-    const info: KeyInfo = {
-      key: process.env.HUB_API_KEY,
-      secret: process.env.HUB_API_SECRET,
-    };
-
-    const remote = await db.remote.setKeyInfo(info);
-
-    await remote.authorize(identity);
-
```
Comment (statictype) [inline src/db/util/init.ts:107]: this seems unrelated to the webpack change. let's at least document it in the PR description

File: src/ui/components/Transactions.tsx [hunk 2]
```diff
@@ -2,62 +2,54 @@
...
 export function Transactions({
   dismiss,
   txs,
 }: React.HTMLAttributes<HTMLDivElement> & TransactionsState) {
-  const Notifications: JSX.Element[] = [];
-  for (const id in txs) {
-    const { extrinsic, status, events } = txs[id] || {};
-    const isComplete = status === 'error' || status === 'success';
+  return (
+    <div className={classes('z-10 fixed right-3 top-3 w-80')}>
```
Comment (statictype) [inline src/ui/components/Transactions.tsx:14]: what is the reason behind this change?

File: src/ui/index.tsx [hunk 3]
```diff
@@ -17,6 +17,8 @@ import {
   NotFound,
 } from 'ui/pages';
 
+globalThis.Buffer = Buffer;
```
Comment (statictype) [inline src/ui/index.tsx:20]: isn't `buffer` part of the node.js polyfills package?

**Review by statictype — CHANGES_REQUESTED** [review]

File: src/ui/components/Transactions.tsx (hunk 2 above)
Comment (kwingram25) [reply src/ui/components/Transactions.tsx:14]: It was to remove the spread operator which was not allowed by the default vite settings and also it makes more sense to use reduce in my opinion.

File: src/db/util/init.ts (hunk 1 above)
Comment (kwingram25) [reply src/db/util/init.ts:107]: The @textile/hub library was throwing errors on compilation so I removed it. Another reason to get rid of textile soon is the bundle size.

**Review by statictype — APPROVED** [review]
